cdap-master/src/main/java/co/cask/cdap/master/startup/ConfigurationCheck.java

```diff
@@ -93,12 +93,9 @@ private void checkBindAddresses() {
     // check if service bind addresses are loopback addresses
     Set<String> bindAddressKeys = ImmutableSet.of(
       Constants.AppFabric.SERVER_ADDRESS,
-      Constants.Dataset.Executor.ADDRESS,
       Constants.Stream.ADDRESS,
```

Comment:
On a distributed cluster stream container, log saver service and metadata services run in YARN containers. So there is no point in checking if they have non-loopback addresses configured. The addresses configured will ignored by the services anyway. We'll need to remove those from the list too, right?